data/json/monsters/zed_ferrous.json

```diff
@@ -0,0 +1,278 @@
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "83500 g",
+    "hp": 100,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "brown",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_cut": 0,
+    "vision_night": 3,
+    "harvest": "zombie",
+    "special_attacks": [ { "type": "bite", "cooldown": 5 }, [ "GRAB", 7 ], [ "scratch", 20 ] ],
+    "death_drops": "default_zombie_death_drops",
+    "death_function": [ "NORMAL" ],
+    "burn_into": "mon_zombie_scorched",
+    "upgrades": { "half_life": 15, "into": "mon_zombie_plated" },
+    "flags": [
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "LOUDMOVES",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_zombie_plated",
+    "type": "MONSTER",
+    "name": { "str": "rust plated zombie" },
+    "description": "A slow moving statue of a zombie covered with a scaly layer of rust.  Each step clangs with the sound of metal.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 15,
+    "volume": "92500 ml",
+    "weight": "120 kg",
+    "hp": 170,
+    "speed": 60,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "brown",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 3,
+    "melee_dice_sides": 4,
+    "melee_cut": 0,
+    "armor_bash": 8,
+    "armor_cut": 8,
+    "armor_bullet": 5,
+    "armor_stab": 8,
+    "vision_night": 3,
+    "luminance": 0,
+    "harvest": "zombie",
+    "special_attacks": [ { "type": "bite", "cooldown": 5 }, [ "GRAB", 7 ], [ "scratch", 20 ] ],
+    "attack_effs": [ { "id": "tetanus", "chance": 10 } ],
+    "death_drops": "default_zombie_death_drops",
+    "death_function": [ "NORMAL" ],
+    "burn_into": "mon_zombie_scorched",
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_zombie_urchin",
+    "type": "MONSTER",
+    "name": { "str": "rust urchin" },
+    "description": "A spiked horror of a zombie covered in long rusty spikes.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 10,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 100,
+    "speed": 60,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "brown",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 3,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_cut": 0,
```

Comment:
This one should get some cut damage.

Reply:
Agreed. Any suggestion on the value? The zombie is fairly slow so I'm leaning twords a hight value.

Reply:
Yeah fairly high for a mid tier zombie. I'm thinking 8-12.

Reply:
12 cut it is